src/System.Text.Json/ref/System.Text.Json.cs

```diff
@@ -340,9 +352,12 @@ public sealed partial class JsonSerializerOptions
         public JsonSerializerOptions() { }
         public bool AllowTrailingCommas { get { throw null; } set { } }
         public int DefaultBufferSize { get { throw null; } set { } }
+        public System.Text.Json.Serialization.JsonNamingPolicy DictionaryKeyPolicy { get { throw null; } set { } }
         public bool IgnoreNullValues { get { throw null; } set { } }
         public bool IgnoreReadOnlyProperties { get { throw null; } set { } }
         public int MaxDepth { get { throw null; } set { } }
+        public bool PropertyNameCaseInsensitive { get { throw null; } set { } }
```

Comment:
One last nit/comment on naming :)

I prefer `PropertyNameIngoreCase`, but I don't feel too strongly about it.